radixdlt/src/integration/java/com/radixdlt/consensus/deterministic/ControlledBFTNetwork.java

```diff
@@ -207,5 +208,13 @@ public void sendVote(Vote vote, ECPublicKey leader) {
 		public void committedStateSync(CommittedStateSync committedStateSync) {
 			putMesssage(new ControlledMessage(sender, sender, committedStateSync));
 		}
+
+		@Override
+		public void committedVertex(Vertex vertex) {
```

Comment:
hmm maybe throw an exception or log something if this empty method is used? We don't want to accidentally be using methods that don't do anything? Might lead to unexpected behaviour?

Reply:
in this case, calling this method is okay. it just means that we don't care about these events but I can add comments regarding this